test/service/test_manifest.py

```diff
@@ -1650,13 +1650,12 @@ class TestManifestExpiration(AzulUnitTestCase):
 
     def test_get_seconds_until_expire(self):
         """
-        Verify a header with valid Expiration and LastModified values returns the correct expiration value
+        Verify a header with valid Expiration and LastModified values returns
+        the correct expiration value.
         """
-        margin = ManifestService._date_diff_margin
-        for object_age, expect_error in [(0, False),
-                                         (margin - 1, False),
-                                         (margin, False),
-                                         (margin + 1, True)]:
+        for object_age, expect_error in [(1, False),
```

Comment:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#14line-wrapping-and-indentation

Reply:
Should extract variable to avoid uglyness of hanging indent.